android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/settings/SettingsFragment.java

```diff
@@ -61,11 +70,9 @@ private void openDialog() {
         builder.setTitle(R.string.delete_user_confirmation_title);
```

Comment:
builder.setCancelable(true); is the default value and therefor an unnecessary line